subprojects/docs/docs.gradle

```diff
@@ -450,17 +451,11 @@ task checkstyleApi(type: Checkstyle) {
     reports.xml.destination = file("$checkstyle.reportsDir/checkstyle-api.xml")
 }
 
-task userguideFragmentSrc(type: UserGuideTransformTask, dependsOn: [userguideStyleSheets]) {
```

Comment:
We used `installation.xml` in two places: Once for the `Installing Gradle chapter` in the userguide and once for the getting starting html file in the distribution. This task created the getting started file (named `installation.xml`, too) from `installation.xml`. Now we have the getting started in a separate asciidoc, so no need anymore for this task.